arangod/VocBase/Identifiers/RevisionId.cpp

```diff
@@ -93,6 +93,14 @@ void RevisionId::toPersistent(std::string& buffer) const {
   rocksutils::uint64ToPersistent(buffer, id());
 }
 
+/// @brief create a revision id with a lower-bound HLC value
+RevisionId RevisionId::lowerBound() { 
+  // "2020-01-01T00:00:00.000Z" => 1577836800000 milliseconds since the epoch
+  RevisionId value{uint64_t(1577836800000) << 20ULL};
```

Comment:
This will be used as an HLC value, and in our HLC implementation the highest 44 bits contain the timestamp, and the lowest 20 bits contain a counter value in case we need multiple values for the same timestamp.